### ISSUE TITLE
Strings Utility Module

### ISSUE THREAD

**Issue by Batmanabcdefg** [JuliaFEM/UMAT.jl#12]
Dear all,

we may include "**Strings Utility Module**" for gurson_porous_plasticity.f90?

Warm regards.

**Comment by TeroFrondelius** [comment]
Dear @OVGULIU, I don't quite understand. Can you rephrase what you mean? Also, if you can be more specific for example point the lines from the file you are referring.

**Comment by Batmanabcdefg** [comment]
@TeroFrondelius if we want to compare Abaqus results, we have to add line code #include 'strings_module.f90'  in gurson_porous_plasticity.f90, is it correct?

**Comment by TeroFrondelius** [comment]
@OVGULIU I don't know. @jvaara do you remember? 

Anyway, @OVGULIU if you will figure it out and will eventually produce comparison results I would appreciate a pull request a lot that we can compare against Abaqus results. 

Similar output as here: https://github.com/JuliaFEM/Materials.jl/blob/master/test/test_chaboche/unitelement_results.rpt (also in similar folder `test/test_gurson`)

**Comment by Batmanabcdefg** [comment]
@TeroFrondelius  we may compare the simulation time in Abaqus and Julia by using the same benchmark or?

**Comment by TeroFrondelius** [comment]
Yes, why not. Most probably Abaqus will be faster than JuliaFEM. In JuliaFEM we currently only have small strains implemented.

**Comment by Batmanabcdefg** [comment]
I would like to check the JuliaFEM 's performance when we use the same Abaqus .inp  & umat & cpus.

**Comment by ahojukka5** [comment]
You probably find out that most of the solution time, the code is doing sparse Cholesky factorization, and it's scaling better with respect to the number of threads in ABAQUS than in Julia. Anyway, this would be a very interesting benchmark and hopefully, you can share some results.
